lintro/ai/prompts/_loader.py

```
@@ -0,0 +1,46 @@
+    Raises:
+        ValueError: If no path parts are supplied.
+        FileNotFoundError: If the resolved template does not exist.
+    """
+    if not path_parts:
+        raise ValueError("load_prompt_template requires at least one path part")
+    resource = files(_TEMPLATES_PACKAGE).joinpath(*path_parts)
+    if not resource.is_file():
+        joined = "/".join(path_parts)
+        raise FileNotFoundError(
+            f"Prompt template not found: {joined} " f"(package {_TEMPLATES_PACKAGE})",
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)